src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -765,7 +783,18 @@ private List<FilterEntry> InitializeNonHostFilter()
             nonHostFilter.ForEach(f => f.PropertyChanged += filter_PropertyChanged);
 
             return nonHostFilter;
+        }
+
+        private List<FilterEntry> InitializeCompatibilityFilter()
+        {
+            var compatibilityFilter = new List<FilterEntry>() { new FilterEntry(Resources.PackageCompatible, Resources.PackageFilterByCompatibility, Resources.PackageCompatibleFilterTooltip, this),
+                                                          new FilterEntry(Resources.PackageUnknownCompatibility, Resources.PackageFilterByCompatibility, Resources.PackageUnknownCompatibilityFilterTooltip, this),
+                                                          new FilterEntry(Resources.PackageIncompatible, Resources.PackageFilterByCompatibility, Resources.PackageIncompatibleFilterTooltip, this)
+            };
 
+            compatibilityFilter.ForEach(f => f.PropertyChanged += filter_PropertyChanged);
```

Comment:
I remember anonymous subscription may cause some difficulties on ubsubscription?

Reply:
Hi @QilongTang - your comment was spot on! While this is the design patter we use for all other filters, prompted by your comment, I added unsubscribing of said events in the Dispose() routine.